common/src/main/java/net/neoforged/gradle/common/CommonProjectPlugin.java

```diff
@@ -116,7 +116,7 @@ public void apply(Project project) {
 
         project.getRepositories().maven(e -> {
             e.setUrl(UrlConstants.MOJANG_MAVEN);
-            e.metadataSources(MavenArtifactRepository.MetadataSources::artifact);
+            e.metadataSources(MavenArtifactRepository.MetadataSources::mavenPom);
```

Comment:
This is the fix for the whole problem with `libraries.minecraft.net` containing artifacts that aren't present on Maven Central (for GAV coordinates that are also present on Central), correct?

This should be mentioned in the PR description at minimum, and possibly in the final squash commit's message.